packages/wrangler/src/__tests__/cloudchamber/create.test.ts

```diff
@@ -10,6 +12,58 @@ import { runWrangler } from "../helpers/run-wrangler";
 import { mockAccount, setWranglerConfig } from "./utils";
 import type { SSHPublicKeyItem } from "../../cloudchamber/client";
 
+const MOCK_DEPLOYMENTS_COMPLEX_RESPONSE = `
```

Comment:
Reviewer note: This is just hoisting reused code up to the top to DRY it up a bit.